test/integration/transactions/transactions.prose.test.ts

```diff
@@ -0,0 +1,73 @@
+import { expect } from 'chai';
+import * as semver from 'semver';
+
+import { type MongoClient } from '../../mongodb';
+
+const metadata: MongoDBMetadataUI = {
+  requires: {
+    topology: '!single'
+  }
+};
+
+describe('Transactions Spec Prose', function () {
+  let client: MongoClient;
+  const started = [];
```

Comment:
Do you think it is worth updating the spec with the additional requirements you found here?  Looks like Python's implementation has found the same things (they're the only other driver that has implemented the test so far).

1. requires transactions (4.0+, non-sharded.  4.2+, sharded)
2. you must create the collection / ensure it exists beforehand

Reply:
Sure https://github.com/mongodb/specifications/pull/1777